main.tf

```diff
@@ -1,97 +1,92 @@
+
 # module "provider" {
 #   source = "./provider/scaleway"
 
 #   organization    = "${var.scaleway_organization}"
 #   token           = "${var.scaleway_token}"
-#   hosts           = "${var.hosts}"
+#   hosts           = "${var.node_count}"
 #   hostname_format = "${var.hostname_format}"
 #   region          = "${var.scaleway_region}"
 # }
 
 module "provider" {
   source = "./provider/hcloud"
 
-  hosts           = "${var.hosts}"
+  hosts           = "${var.node_count}"
   token           = "${var.hcloud_token}"
   type            = "${var.hcloud_type}"
   ssh_keys        = "${var.hcloud_ssh_keys}"
   location        = "${var.hcloud_location}"
   hostname_format = "${var.hostname_format}"
 }
 
-/*module "provider" {
-  source = "./provider/digitalocean"
+# module "provider" {
```

Comment:
can you put a comment here on why these sections are proposed to be commented out?

Reply:
I commented out cloudflare, I figure people getting started are more likely to have only the hosting account. Since it appears that main.tf is meant to be used for switching selections on and off, I thought this was a reasonable starting state.

Other sections were left the same. I changed block comments to line comments, IME it's easier in most editors to comment/uncomment sections if they're line comments.